features/netsocket/PPPInterface.cpp

```diff
@@ -92,6 +92,41 @@ const char *PPPInterface::get_ip_address()
     return NULL;
 }
 
+nsapi_error_t PPPInterface::get_ip_address(SocketAddress *address)
+{
+    if (address) {
```

Comment:
Sorry, I didn't notice this earlier, but there's a bug in here. Should be `if (!address) {`

Reply:
Thanks for pointing this out, @kivaisan .
PR that addresses this is redy: https://github.com/ARMmbed/mbed-os/pull/11975 . @0xc0170 , sorry for this disturbance, but could we merge that PR, so the release has bug free code?

The reason why this was not spotted by CI is because the tests are pending: https://github.com/ARMmbed/mbed-os/pull/11878